sonarapp/Program.cs

```diff
@@ -18,6 +18,8 @@ static void Main(string[] args)
 
             Boolean whatiffff = (new verify()).verifywhat("whatififff");
 
+            Boolean whatifsssfff = (new verify()).verifywhat("whatififfsssf");
+
         }
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused 'whatifsssfffjenkin' local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=csharpsquid%3AS1481)